contrib/pyln-spec/requirements.txt

```diff
@@ -1 +1 @@
-pyln.proto.message
+pyln.proto==0.8.4
```

Comment:
This version doesn't exist on the test or prod PyPI, so installing this package with this requirement would fail.

Reply:
Yes, but it will once this is merged :)

It's a real dependency, since this needs the latest message module.